adding-a-new-module/template/build.gradle.kts

```diff
@@ -0,0 +1,47 @@
+plugins {
+    kotlin("multiplatform")
+    id("jacoco")
+    id("maven-publish")
+    id("com.android.library")
+    id("org.jlleitschuh.gradle.ktlint")
+}
+
+val ext = (gradle as ExtensionAware).extra
+
+apply(from = "../gradle/publishable_component.gradle")
+
+group = "com.splendo.kaluga"
+version = ext["library_version"]!!
+
+repositories {
+    google()
+    mavenCentral()
+}
```

Comment:
also in `components.gradle`, can probably be cleaned up from modules where it still is added.

Reply:
Yep. But I would prefer to do it in separate PR (clean other projects)

Reply:
Good idea to do those separate, but can be removed from the template.